incremental-font-transfer/src/patchmap.rs

```diff
@@ -1583,74 +1587,69 @@ mod tests {
         assert_eq!(expected, patches);
     }
 
-    fn check_uri_template_substitution(template: &str, value: u32, expected: &str) {
+    fn check_url_template_substitution(template: &[u8], value: u32, expected: &str) {
         assert_eq!(
-            PatchUri::expand_template(template, &PatchId::Numeric(value))
+            PatchUrl::expand_template(template, &PatchId::Numeric(value))
                 .unwrap()
                 .as_ref(),
             expected,
         );
     }
 
-    fn check_invalid_uri_template_substitution(template: &str, value: u32) {
-        assert_eq!(
-            PatchUri::expand_template(template, &PatchId::Numeric(value)),
-            Err(UriTemplateError)
-        );
-    }
-
-    fn check_string_uri_template_substitution(template: &str, value: &str, expected: &str) {
+    fn check_string_url_template_substitution(template: &[u8], value: &str, expected: &str) {
         assert_eq!(
-            PatchUri::expand_template(template, &PatchId::String(Vec::from(value.as_bytes())))
+            PatchUrl::expand_template(template, &PatchId::String(Vec::from(value.as_bytes())))
                 .unwrap()
                 .as_ref(),
             expected,
         );
     }
 
     #[test]
-    fn uri_template_substitution() {
+    fn url_template_substitution() {
         // These test cases are used in other tests.
-        check_uri_template_substitution("//foo.bar/{id}", 1, "//foo.bar/04");
-        check_uri_template_substitution("//foo.bar/{id}", 2, "//foo.bar/08");
-        check_uri_template_substitution("//foo.bar/{id}", 3, "//foo.bar/0C");
-        check_uri_template_substitution("//foo.bar/{id}", 4, "//foo.bar/0G");
-        check_uri_template_substitution("//foo.bar/{id}", 5, "//foo.bar/0K");
+
+        let foo_bar_id = &[
+            10, b'/', b'/', b'f', b'o', b'o', b'.', b'b', b'a', b'r', b'/', 128,
```

Comment:
yea i think all of these would be easier to understand as either a byte string literal or with a macro (to handle the prefix/suffix)